MekHQ/src/mekhq/campaign/parts/AmmoStorage.java

```diff
@@ -283,7 +283,6 @@ public Object getNewEquipment() {
 
     @Override
     public String failToFind() {
-        resetDaysToWait();
```

Comment:
The AmmoStorage, Armor, and Refit parts objects all add `resetDaystoWait()` to the `failToFind` method. However, this is redundant, because each of the `Campaign.goShopping` methods already does this reset at the proper place for all IAcquisitionWork. In this case, it was causing some issues with planetary acquisition because we don't want to reset that until we have failed to find stuff on all the planets on our list.